vowpalwabbit/parse_primitives.h

```diff
@@ -160,3 +160,22 @@ inline int int_of_string(VW::string_view s)
 
   return i;
 }
+
+inline int int_of_string_ml(VW::string_view s)
```

Comment:
What does `ml` mean?

Reply:
`ml` in `int_of_string_ml` represents multiclass here. As multilabel can have multiple labels, so the error should only invoke in multilabel one. Hence I made another function to get int from the string.